webpack/InsightsCloudSync/InsightsCloudSync.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { Button } from 'patternfly-react';
+import { syncInsights } from './InsightsCloudSyncActions';
+
+const InsightsCloudSync = () => (
+  <IntlProvider locale={navigator.language}>
+    <div className="insights-cloud-sync">
+      <h1>{__('Red Hat Insights Sync')}</h1>
+      <Button bsStyle="primary" onClick={syncInsights()}>
+        {__('Sync now')}
+      </Button>
```

Comment:
I think we should move it into its own component folder,
and `cconnect` that component to `redux` so we could pass the action properly

Reply:
can you move it into another component?
e.g: `webpack/InsightsCloudSync/SyncButton`